packages/query/src/block-processor.ts

```diff
@@ -162,12 +162,12 @@ export class BlockProcessor implements BlockProcessorInterface {
 
     // special case genesis sync
     if (currentHeight === -1n) {
-      // initialize validator info at genesis
-      // TODO: use batch endpoint https://github.com/penumbra-zone/penumbra/issues/4688
+      // create first epoch and initialize validator info at genesis
+      await this.indexedDb.addEpoch(currentHeight + 1n);
```

Comment:
suggestion: feels more clear if we just do:

```
await this.indexedDb.addEpoch(0n);
void this.updateValidatorInfos(0n);
```

Reply:
good observation, changed